PRIVATE_SYNTAX_FAQ.md

```diff
@@ -1,12 +1,36 @@
-## Why aren't declarations `private x`?
+## What's this about `#`?
+
+`#` is the new `_`. Whereas
```

Comment:
I feel like this slogan requires some sort of exclamation point or emoji or something, and being presented on its own line.